src/utils/formatUtil.ts

```diff
@@ -232,7 +232,9 @@ export function createAnnotatedPath(
   const { rootFolder = 'input', subfolder } = options
   if (typeof item === 'string')
     return `${createPath(item, subfolder)}${createAnnotation(item, rootFolder)}`
-  return `${createPath(item.filename ?? '', item.subfolder)}${item.type ? createAnnotation(item.type, rootFolder) : ''}`
```

Comment:
I think folding this expression introduces extra `\n's to the string expression. Please revert.